changelog.d/20230324_162931_Gavin.Huttley.md

```diff
@@ -0,0 +1,44 @@
+<!--
+A new scriv changelog fragment.
+
+Uncomment the section that is right (remove the HTML comment wrapper).
+-->
+
+<!--
+### Contributers
+
+-   A bullet item for the Contributers category.
+
+-->
+<!--
+### ENH
+
+-   A bullet item for the ENH category.
+
+-->
+<!--
+### BUG
+
+-   A bullet item for the BUG category.
+
+-->
+<!--
+### DOC
+
+-   A bullet item for the DOC category.
+
+-->
+<!--
+### Deprecations
+
+-   Removed the original Composable app classes and related decorators for
+    user based apps. `user_function` and `appify` are replaced by the
```

Comment:
We can add here:
Removed the original Composable app classes and related decorators for user base apps and replaced with `define_app` decorator.

Reply:
And what about _checkpointable class?

Reply:
Your suggested additional text exists in the file, not sure why github not showing it...

`_checkpointable` is a private class, no-one should have been using it